src/conf_mode/interfaces-bridge.py

```diff
@@ -99,7 +73,7 @@ def get_config(config=None):
         # the default dictionary is not properly paged into the dict (see T2665)
         # thus we will ammend it ourself
         default_member_values = defaults(base + ['member', 'interface'])
-        vlan_aware = False 
+        vlan_aware = True if 'enable_vlan' in bridge else False
```

Comment:
Why is this variable assignment required? You only reference it once in line 100 and you could combine the statements.

We always use this pattern: `if 'foo' in bar:` stick to the pattern!